### PR TITLE
Fixed typo and added Links

### PR DESCRIPTION
**Pull request by Maurice22**
Fixed a typo and added links to respective sites.

### CODE REVIEW

File: src/components/Dashboard.js [hunk 1]
```diff
@@ -63,13 +63,17 @@ export default function Dashboard() {
     <h1 className="font-teaser py-4 pt-10">Deine Hilfegesuche</h1>
 
     {entries.length === 0
-      ? <div className="font-open-sans">Du hast keine Hilfegesuche eingestellt.</div>
+      ? <div className="font-open-sans">
+          Du hast noch keine Hilfegesuche eingestellt. Du kannst ein neues Gesuch <Link class="underline" to="/ask-for-help" onClick={() => fb.analytics.logEvent('button_want_to_help')}>hier</Link> erstellen.
```
Comment (florianschmidt1994) [inline src/components/Dashboard.js]: `<Link class=...` should be `<Link className=...`

File: src/components/Dashboard.js [hunk 2]
```diff
@@ -63,13 +63,17 @@ export default function Dashboard() {
...
+        </div>
       : entries.map(entry => (<Entry {...entry} key={entry.id} owner={true}/>))}
 
     <h1 className="font-teaser py-4 pt-10">Deine Benachrichtigungen </h1>
 
     {offers.length === 0
-      ? <div className="font-open-sans">Du hast keine Hilfegesuche eingestellt.</div>
+      ? <div className="font-open-sans">
+          Du hast noch keine Benachrichtigungen aktiviert. Du kannst neue Benachrichtigungen <Link class="underline" to="/notify-me" onClick={() => fb.analytics.logEvent('button_want_to_help')}>hier</Link> registrieren.
```
Comment (florianschmidt1994) [inline src/components/Dashboard.js]: `<Link class=...` should be `<Link className=...`

**Comment by Maurice22** [comment]
Done

**Comment by florianschmidt1994** [comment]
Sweet! LGTM

**Pull request merged**